docs/wagtail-pages.md

```diff
@@ -0,0 +1,207 @@
+```
+
+## StreamFields
+
+StreamFields are special Django model fields provided by Wagtail for 
+[freeform page content](https://docs.wagtail.io/en/v1.13.4/topics/streamfield.html). 
+They allow a content editor to pick any number number of optional components
+and place them in any order within their StreamField. 
+In practice this provides the 
+[flexibilty of a large rich text box with the structure of individual components](https://torchbox.com/blog/rich-text-fields-and-faster-horses/).
+
+For example, our [`LandingPage` page model includes a `header` StreamField](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/landing_page.py#L13) that can have a hero and/or a text introduction:
+
+```python
+from wagtail.wagtailcore.fields import StreamField
+
+from v1.atomic_elements import molecules
+from v1.models import CFGOVPage
+
+
+class LandingPage(CFGOVPage):
+
+    header = StreamField([
+        ('hero', molecules.Hero()),
+        ('text_introduction', molecules.TextIntroduction()),
+    ], blank=True)
+```
+
+**Insert link to StreamField docs here.**
+
+## Panels
+
+Editor panels define how the page's [fields](#fields) and [StreamFields](#streamfields) will be organized for content editors.
+
+The base Wagtail `Page` class and the [`v1.models.base.CFGOVPage` class](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/base.py#L60) 
+defines a specific set of panels to which all fields should be added:
+
+- `content_panels`. 
+  These fields appear on the "General Content" tab when editing a page. 
+  For page body content.
+- [`sidefoot_panels`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/base.py#L114-L116).
+  These fields appear on the "Sidebar" tab when editing a page. 
+- [`settings_panels`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/base.py#L118-L125).
+  Appears on the "Configuration" tab when editing a page. 
+  Page configuration such as the categories, tags, scheduled publishing, etc.
+
+Most fields will simply require a `FieldPanel` to be added to one of the sets of panels above. 
+StreamFields will require a `StreamFieldPanel`. 
+See the [Wagtail documentation for additional, more complex panel options](https://docs.wagtail.io/en/v1.13.4/topics/pages.html#editor-panels).
+
+For example, in our `BrowsePage` (used in the [database fields example above](#fields),
+[the `secondary_nav_exclude_sibling_pages` `BooleanField` is added to the `sidefoot_panels` as a `FieldPanel`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/browse_page.py#L63):
+
+```python
+from django.db import models
+from wagtail.wagtailadmin.edit_handlers import FieldPanel
+
+from v1.models.base import CFGOVPage
+
+
+class BrowsePage(CFGOVPage):
+    secondary_nav_exclude_sibling_pages = models.BooleanField(default=False)
+
+    # …
+
+    sidefoot_panels = CFGOVPage.sidefoot_panels + [
+        FieldPanel('secondary_nav_exclude_sibling_pages'),
+    ]
+```
```

Comment:
It would be helpful to have a sentence here about what this actually does, e.g. "This creates a checkbox title 'Secondary nav exclude...' on the "Sidebar" tab in the Wagtail admin. Checking or unchecking this box sets the value of the `secondary_nav_exclude_sibling_pages` field when the page model is saved to the database."